File: Command/Python/src/executors/stereo.py
class Stereo:

    def __init__(self, name: str) -> None:
        self._name = f"{name} stereo"
        self.speed = self.Speeds.HIGH

    @property
    def name(self) -> str:
        return self._name

    @name.setter
    def name(self) -> None:
        raise AttributeError("'name' is read only")

    def on(self) -> None:
        print(f"{self._name} is on")

    def off(self) -> None:
        print(f"{self._name} is off")

    def set_cd(self) -> None:
        print(f"{self._name} is set for CD input")

    def set_dvd(self) -> None:
        print(f"{self._name} is set for DVD input")

    def set_radio(self) -> None:
        print(f"{self._name} is set to radio")

    def set_volume(self, volume: int) -> None:
        print(f"{self._name} volume is set to {volume}")

File: Command/DotNet/Command/executors/Stereo.cs
namespace CommandPattern.Executors;


public class Stereo
{
    public string Name { get; set; }

    public Stereo(string name)
    {
        this.Name = $"{name} stereo";
    }

    public void On()
    {
        Console.WriteLine($"{this.Name} is on");
    }

    public void Off()
    {
        Console.WriteLine($"{this.Name} is off");
    }

    public void SetCd()
    {
        Console.WriteLine($"{this.Name} is set for CD input");
    }

    public void SetDvd()
    {
        Console.WriteLine($"{this.Name} is set for DVD input");
    }

    public void SetRadio()
    {
        Console.WriteLine($"{this.Name} is set to radio");
    }

    public void SetVolume(int volume)
    {
        Console.WriteLine($"{this.Name} volume set to {volume}");
    }
}